### ISSUE TITLE
Discussion: The Windows and Linux download links are broken

### ISSUE THREAD

**Issue by darren277** [facebook/flipper#5517]
Neither the Windows nor the Linux download links work for the latest build of the Flipper desktop application.

(The links from here: https://fbflipper.com/docs/getting-started/)

**Comment by Sopkow** [comment]
Huge blocker for my team, not sure why more than half of all users are being ignored at the moment?

**Comment by passy** [comment]
We have updated it. At this point, the links should point to the latest release again.

**Comment by Sopkow** [comment]
@passy Not sure where the links were updated? Still seeing broken links on the setup page

![image](https://github.com/facebook/flipper/assets/31808798/57b8ed8e-40a0-4ae2-a505-88824865c8c5)

**Comment by darren277** [comment]
> @passy Not sure where the links were updated? Still seeing broken links on the setup page

Hmm, yeah, those download links are still broken.

@passy, is there another place where there are links for the compiled desktop app to be downloaded?

**Comment by passy** [comment]
@Sopkow Thanks. I didn't see these links before. We currently do not have pre-built apps for Linux and Windows and sadly no capacity to fix this in the near future. If someone would like to pick this up and add it to our GitHub Actions config, I'd be happy to review the PR.

**Comment by itjuh** [comment]
This is the download link for the most recent Windows version file.
[flipper-win](https://github.com/facebook/flipper/releases/download/v0.239.0/Flipper-win.zip)

**Comment by itjuh** [comment]
I will share a link to the version I am using. Use it temporarily.
[flipper-win(v0.220.0)]([https://github.com/facebook/flipper/releases/download/v0.220.0/Flipper-win.zip]

**Comment by Nitrammets** [comment]
Makes me feel good that even facebook can't handle it's releases... Still broken btw

**Comment by masztalski** [comment]
@passy still windows link points to tgz file instead of zipped built app. On Flipper 0.239.0 I don't see even any update info available

**Comment by JacktheGodzillaSlayer** [comment]
yes, on the Flipper landing page the link on windows points to the same as Linux, which is ".../other"

**Comment by RCG-RCorrales** [comment]
Issue is still ongoing - links to *other*.
